src/hooks/useBookmarkAutoPronounce.js

```diff
@@ -8,13 +8,15 @@ import {
 export default function useBookmarkAutoPronounce() {
   const [currentPronouncingState, setCurrentPronouncingState] = useState();
   const [currentPronouncingString, setCurrentPronouncingString] = useState();
+  const [isPronounceBookmark, setIsPronounceBookmark] = useState();
```

Comment:
Can you add a comment to this `isPronounceBookmark`? I don't understand it from it's name.
It seems to suggest that we might have some `pronounce` bookmarks, and some `non-pronounce` ones. But I think it's semantics is more `isPronounciationEnabled`? But then, why do we have the `currentPronouncingState`? 

Maybe a little explanation at the top of the hook?

Reply:
I have renamed it to AutoPronounce Bookmark and decided to remove the currentPronouncingState value as it is not used outside. This was what was given rise to the bug, since before I was comparing it on the exercises/nextNav component.